Public/Src/Tools/Execution.Analyzer/Analyzers.Core/Xldb/XLGToDBAnalyzer.cs

```diff
@@ -267,7 +285,6 @@ public override bool CanHandleEvent(Scheduler.Tracing.ExecutionEventId eventId,
 

```

Comment:
I don't think `size` has any meaning in xldb world - you are expanding all the paths...

Reply:
It doesn't, and I added it to mimic the behavior of event stats from before.

---
In reply to: [319314407](https://github.com/microsoft/BuildXL/pull/801#discussion_r319314407) [](ancestors = 319314407)